src/main/java/lv/ctco/javaschool/vote/boundary/ResponseWrapper.java

```diff
@@ -0,0 +1,18 @@
+package lv.ctco.javaschool.vote.boundary;
+
+import javax.ws.rs.core.Response;
+
+public class ResponseWrapper {
+
```

Comment:
Please append "Response" to the names of these three methods to make them clearer. "getBadRequestResponse" IMO is better than "getBadRequest".